app/components/Header/MobileHeader.tsx

```diff
@@ -0,0 +1,72 @@
+"use client";
+import Image from "next/image";
+import Link from "next/link";
+
+import { useState } from "react";
+
+export function MobileHeader() {
+  const [navOpen, setNavOpen] = useState(false);
+  function toggleNav() {
+    setNavOpen(!navOpen);
+  }
+  return (
+    <header className="flex justify-between items-center sm:hidden bg-orange">
+      <div className="flex items-center gap-2">
+        <Image
+          src={"/misc/chillNoBG.png"}
+          height={50}
+          width={50}
+          alt="hamburger"
+          className="h-auto"
+        />
+        <h1>Thomas Burger</h1>
+      </div>
+      <div onClick={toggleNav}>
+        {navOpen ? (
```

Comment:
Let's decouple the nav from the state of the menu icon.

Reply:
That makes sense. What would you like me to attach it to?